docs/03-apis/02-collections.md

```diff
@@ -0,0 +1,19 @@
+# Collections
+
+Collections are a way of organising images in Grid. Collections have a hierarchy similar to:
+
+```
+ROOT COLLECTION 1
+  ⎿ Collection A
+    ⎿ Collection B
+       ⎿ ...
+ROOT COLLECTION 2
+  ⎿ Collection C
+    ⎿ Collection D
+       ⎿ ...
+```
+
+## Root collections
+Currently, root collections can only be created via the API. That is, Kahuna doesn't have a UI for it.
+
+We can use [create-root-collection.sh](../../dev/script/create-root-collection.sh) to create a root collection.
```

Comment:
Is it just a case of running the script with a single argument: the name of the collection?

Reply:
Yep. Maybe this should be more explicit here too.

Reply:
Done.